### PR TITLE
Add Stride game engine

### PR DESCRIPTION
**Pull request by manio143**
### SteamDB app page links to a few games using this
https://steamdb.info/app/1256380/
https://steamdb.info/app/1531540/

### Brief explanation of the change
Adding a reference to the Stride game engine. Currently Stride is getting a lot of traction and we're looking to see if there's more games out there deployed to Steam that use Stride.
See https://github.com/stride3d/stride

### CODE REVIEW

File: rules.ini [hunk 1]
```diff
@@ -11,6 +11,7 @@ Build[] = (?:^|/)COMMIT\.DAT$
 Build[] = (?:^|/)GAME\.CON$
 CFG = \.cfg$ ; idTech1
 DATA_WIN = (?:^|/)data\.win$ ; GameMaker
+DATA_DB_INDEX = (?:^|/)data/db/index$ ; Stride
```
Comment (Lifeismana) [inline rules.ini:14]:
```suggestion
```
detecting the engine via its dll is enough, we don't need evidences here

File: descriptions/Engine.Stride.md [hunk 2]
```diff
@@ -0,0 +1 @@
+[**Stride**](https://www.stride3d.net/) is a free and open-source game engine written in C#. Stride is a member of .NET Foundation. It was originally developed by Silicon Studio under the name Xenko and has been open-sourced in 2018 and renamed to Stride in 2020.
```
Comment (Lifeismana) [inline descriptions/Engine.Stride.md:1]:
```suggestion
[**Stride**](https://www.stride3d.net/) is a free and open-source game engine written in C#. It was originally developed by Silicon Studio under the name Xenko and has been open-sourced in 2018 and renamed to Stride in 2020.
```
I'm sure you are proud of that but it's not really needed here

File: rules.ini [hunk 3]
```diff
@@ -128,6 +129,8 @@ Snowdrop = \.sdfdata$
 Solar2D = (?:^|/)CoronaLabs\.Corona\.Native\.dll$
 Source = (?:^|/)(?:vphysics|bsppack)\.(?:dylib|dll|so)$
 Source2 = (?:^|/)gameinfo\.gi$
+Stride[] = (?:^|/)Stride\.Core\.dll$
+Stride[] = (?:^|/)Xenko\.Core\.dll$
```
Comment (Lifeismana) [inline rules.ini:133]:
```suggestion
Stride = (?:^|/)(?:Stride|Xenko)\.Core\.dll$
```

**Review by Lifeismana — CHANGES_REQUESTED** [review]

File: rules.ini (hunk 1 above)
Comment (manio143) [reply rules.ini:14]: I'm ok to remove it. However, people may use a single file publishing method that would pack all the DLLs into the executable. In that case there is no DLLs to detect. But content files would still be separately included in the `data` folder.

Comment (manio143) [reply rules.ini:13]: @Lifeismana waiting for a comment from you, should I nonetheless remove this?

File: tests/types/Evidence.DATA_DB_INDEX.txt [hunk 4]
```diff
@@ -0,0 +1,3 @@
+data/db/index
+/data/db/index
+Folder/data/db/index
```
[the remark this answers is not in the document]
Comment (manio143) [reply tests/types/Evidence.DATA_DB_INDEX.txt]: Applied

**Comment by manio143** [comment]
@Lifeismana applied all your requested changes. Looking for you to merge this.

**Pull request merged**